### PR TITLE
WQP-1411 - Result (result_no_source)

### PR DESCRIPTION
**Pull request by dsteinich**

### CODE REVIEW

File: src/main/java/gov/acwi/wqp/etl/DbConfig.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		return dataSourceProperties().initializeDataSourceBuilder().build();
+	}
+
+	@Bean
+	@Primary
+	public JdbcTemplate jdbcTemplate() {
+		return new JdbcTemplate(dataSource());
+	}
+
+	@Bean
```
Comment (mbucknell) [inline src/main/java/gov/acwi/wqp/etl/DbConfig.java]: Why is this needed? I don't see this source in the src/main/resources/application.yml

Comment (dsteinich) [reply src/main/java/gov/acwi/wqp/etl/DbConfig.java]: Good catch, I'll take it out
